### DISCUSSION TITLE
Vercel's main website has a bug

### DISCUSSION THREAD

**Discussion by nitlix** [vercel/community#4731] [Bug Report]
### Page to Investigate

https://vercel.com/home

### Steps to Reproduce

Simply hover on the "Features" text and the box will be expanding itself forever.

![image](https://github.com/vercel/community/assets/66027553/2093aa27-e1dc-45a8-8ea0-31863be9bf77)

**Comment by amyegan** [comment]
Thanks for reporting this! I'm not seeing it myself in Firefox, Chrome, or Safari. Can you tell me more about your browser, or anything else I should know, so I can look into this more?

**Reply by nitlix** [reply]
I've got the issue on opera one; I've also attached a video showcasing the bug 👍:
https://github.com/vercel/community/assets/66027553/c05dc25f-9891-4979-8b8d-64decb763c86

**Reply by amyegan** [reply]
Perfect example! Thanks for sharing this. I'll pass it along to our website team
